.github/workflows/bundle_with_dakota_macos.yml

```diff
@@ -181,12 +227,27 @@ jobs:
         echo "PATH=${PATH}:${HOME}/${INSTALL_DIR}/bin" >> ${GITHUB_ENV}
         echo "LD_LIBRARY_PATH=${LD_LIBRARY_PATH}:${HOME}/${INSTALL_DIR}/lib" >> ${GITHUB_ENV}
         echo "DYLD_LIBRARY_PATH=${DYLD_LIBRARY_PATH}:${HOME}/${INSTALL_DIR}/lib:$INSTALL_DIR/bin" >> ${GITHUB_ENV}
+
     - name: Checkout Carolina
       uses: actions/checkout@v3
       with:
         fetch-depth: 0
+
     - name: Install Carolina and make wheel
       run: |
+        if [ -d /tmp/build-env ]; then
+          source /tmp/build-env/bin/activate
+        fi
+        python --version
+
+        arch_ver=$(uname -m)
+        export _PYTHON_HOST_PLATFORM="macosx-11.0-$arch_ver"
+        export ARCHFLAGS="-arch $arch_ver"
+
+        if [ "$arch_ver" == "arm64" ]; then
```

Comment:
Suppose this is necessary to make it work, when/if GH actions change their runner/arch combinations, will this still hold up?

Reply:
The tools actually update the version number in the host platform, so this hints to the wheel build that we don't want `universal` wheels, but rather `arm64` or `x86_64` wheels separate.
I will add a comment to this since the next person won't know these details.